pvlib/irradiance.py

```diff
@@ -460,31 +465,31 @@ def poa_components(aoi, dni, poa_sky_diffuse, poa_ground_diffuse):
     ----------
     aoi : numeric
         Angle of incidence of solar rays with respect to the module
-        surface, from :func:`aoi`.
+        surface, from :func:`aoi` [°]. See :term:`aoi`.
 
     dni : numeric
-        Direct normal irradiance (Wm⁻²), as measured from a TMY file or
-        calculated with a clearsky model.
+        Direct normal irradiance [Wm⁻²], as measured from a TMY file or
+        calculated with a clearsky model. See :term:`dni`.
 
     poa_sky_diffuse : numeric
-        Diffuse irradiance (Wm⁻²) in the plane of the modules, as
-        calculated by a diffuse irradiance translation function
+        Diffuse irradiance [Wm⁻²] in the plane of the modules, as
+        calculated by a diffuse irradiance translation function.
 
     poa_ground_diffuse : numeric
-        Ground reflected irradiance (Wm⁻²) in the plane of the modules,
+        Ground reflected irradiance [Wm⁻²] in the plane of the modules,
         as calculated by an albedo model (eg. :func:`grounddiffuse`)
```

Comment:
Usually see square brackets, but this is at the end of the variable description. What's appropriate in the middle of the description?

Reply:
I'd use the same delimiters regardless of position (end or in the middle). I don't know why we started with square brackets; I assumed it had to do with sphinx but that's probably not correct.